test/common.go

```diff
@@ -0,0 +1,131 @@
+package test
+
+import (
+	"bytes"
+	"embed"
+	cmpact "encoding/json"
+	"github.com/goccy/go-json"
+	"fmt"
+	manifestsdk "github.com/TBD54566975/ssi-sdk/credential/manifest"
+	"github.com/oliveagle/jsonpath"
+	"github.com/pkg/errors"
+	credmodel "github.com/tbd54566975/ssi-service/internal/credential"
+	"io"
+	"net/http"
+)
+
+const (
+	endpoint = "http://localhost:8080/"
+	version  = "v1/"
+)
+
+var (
+	//go:embed testdata
+	testVectors embed.FS
+)
+
+func compactJSONOutput(json string) string {
+	jsonBytes := []byte(json)
+	buffer := new(bytes.Buffer)
+	if err := cmpact.Compact(buffer, jsonBytes); err != nil {
+		fmt.Println(err)
+	}
+
+	return buffer.String()
+}
+
+func getJSONElement(jsonString string, jsonPath string) (string, error) {
+	jsonMap := make(map[string]interface{})
+	if err := json.Unmarshal([]byte(jsonString), &jsonMap); err != nil {
+		return "", errors.Wrap(err, "problem with unmarshalling json string")
+	}
+
+	element, err := jsonpath.JsonPathLookup(jsonMap, jsonPath)
+	if err != nil {
+		return "", errors.Wrap(err, "problem with finding element in json string")
+	}
+
+	return element.(string), err
+}
+
+func get(url string) (string, error) {
+	fmt.Printf("\nPerforming GET request to:  %s\n", url)
+
+	resp, err := http.Get(url) // #nosec: testing only.
+	if err != nil {
+		return "", errors.Wrap(err, "problem with finding element in json string")
+	}
+
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return "", errors.Wrap(err, "problem with parsing body")
```

Comment:
Here' and elsewhere. When wrapping errors, the context read's better when you describe what operation lead to the error.
```suggestion
		return "", errors.Wrap(err, "reading all body")
```